js/utilities.js

```diff
@@ -53,18 +38,25 @@
 	var TAB_KEYCODE = 9;
 	var UP_ARROW_KEYCODE = 38;
```

Comment:
https://github.com/ExactTarget/fuelux/pull/1856/files/001230ee21d5c2fe9aa39f9f3ed713508f8bf6be..b4161dd5b6f5987346efa0eaa7e0e362e2b02204#diff-2b43f466b20ec5f1d5c780c93afa403cR68

Move up the const object and use it in the is*Key block. 

```
$.fn.utilities = {
  CONST: CONST,
```